src/pages/projectSettings/Tabs.tsx

```diff
@@ -60,7 +60,9 @@ export const ProjectSettingsTabs: React.FC<Props> = ({
         >
           Save changes on page
         </Button>
-        {!useRepoSettings && <Button>Default to Repo on page</Button>}
+        {projectData && useRepoSettings && (
+          <Button data-cy="default-to-repo">Default to Repo on page</Button>
```

Comment:
Can we use capitalization for buttons/headers (Default to Repo on Page)? sorry i didn't see this when we did "save changes on page" but I think this should extend to that (move to new repo, etc).